.circleci/config.yml

```diff
@@ -57,15 +56,33 @@ jobs:
       - run:
           name: Restore cache
           command: |
-            docker load -i /cache/dev.tar
-            docker load -i /cache/prod.tar
+            docker load -i /tmp/dev.tar
+            docker load -i /tmp/prod.tar
       - run:
           name: Test Code
           command: docker run prio:dev
       - run:
           name: Integration Code
           command: docker run prio:prod scripts/test-cli-integration
   
+  test-batch-example:
+    machine: true
+    working_directory: ~/python-libprio/examples/batched-processing
+    steps:
+      - checkout:
+          path: ~/python-libprio
+      - restore_cache:
+          key: v1-{{.Branch}}
+      - run:
+          name: Restore Docker image cache
+          command: docker load -i /tmp/dev.tar
+      - run:
+          name: Build the compose container
+          command: docker-compose build
+      - run:
+          name: Run the batched-processing integration test
+          command: scripts/integration.sh
```

Comment:
Should this be `make test`?

Reply:
I was being cautious about dependencies because some of the images on CircleCI don't even contain bash. I guess the base machine image has all the build essentials baked in.